ran-core/src/main/java/io/ran/Annotations.java

```diff
@@ -1,3 +1,8 @@
+/* Copyright (C) Persequor ApS - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * Written by Persequor Development Team <partnersupport@persequor.com>, 2022-02-22
```

Comment:
Just want to point out that your copyright date seems to not be automatically set? :sweat_smile:

Reply:
Also since RAN is open-source I don't think that `Proprietary and confidential` is correct?

Reply:
the script looks for first commit i think, was it a rename of the project, so most files seem to have same date? And I just assumed we want the license, since it was on few files, but not all. Maybe we don't want them at all

Reply:
yes, he just did. we should come up with different wording for open source projects

Reply:
[c9ac855](https://github.com/persequor-com/ran/pull/39/commits/c9ac8555dbfcb2974f16afad37215d24ea11efc0) it was in the root